en/docs/manage/api-rate-limit.md

```diff
@@ -0,0 +1,43 @@
+# API Rate-Limiting in Choreo
+
+Real world computation requirements are operating under finite number of computing resources. Considering this, Choreo 
+provides API rate-limiting capabilities to control the traffic to your APIs. Choreo allows you to configure the 
+rate-limiting considering different levels and time windows. By enabling the rate-limiting, you will be able to protect 
+your APIs from malicious invocations and utilize the available computing resources efficiently. This documentation 
+covers following aspects related to the API rate-limiting in Choreo.
+
+  - [API level rate-limiting](#api-level-rate-limiting)
+  - [Operation level rate-limiting](#operation-level-rate-limiting)
+  - [Response headers related to the rate-limiting](#response-headers-related-to-rate-limiting)
+
+To enable rate-limiting for a given API, you need to select the relevant API component and navigate to the **Manage** 
```

Comment:
Shall we have a subheading as `Enable rate limiting for an API` and include the instructions as a numbered set of steps?

Reply:
Generally, you need the hyphen in rate limiting only if it functions as an adjective before the noun it describes. If the noun comes first, you can leave the hyphen out.